wordle-web/src/components/ValidWordList.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-row justify="center">
+    <v-dialog v-model="dialog" scrollable width="auto"
+      ><template v-slot:activator="{ props }">
+        <v-btn color="primary" v-bind="props" @click="props">{{
+          `(${items.length}) Possible Words`
+        }}</v-btn>
+      </template>
+      <v-card>
+        <v-card-title>Select Words</v-card-title>
+        <v-divider></v-divider>
+        <v-card-text style="height: 300px; max-width: 150px">
+          <v-hover
+            ><v-btn
```

Comment:
```suggestion
            > <v-btn
```
If you add spaces between these, your linter won't do this weird formatting.